folium/features.py

```diff
@@ -652,6 +682,70 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
         self.className = class_name
 
 
+class Tooltip:
```

Comment:
My guess is that deriving from `object` is proffered, no? (This is the old style `class` if I'm not mistaken.)

Reply:
Is there a reason to use the new style if it does not need class inheritance from another property? Or should I instantiate it with no arguments like `class Tooltip():`?